docker/service/Dockerfile

```diff
@@ -1,8 +1,11 @@
 # Builder part for maven packaging.
-FROM maven:3-openjdk-11 as builder
+FROM eclipse-temurin:17 as builder
```

Comment:
Can perhaps use `maven:3-eclipse-temurin-17` for this one, that's what we use in our jenkinsfiles as well.